### PR TITLE
Update Nagios check

### PR DESCRIPTION
**Pull request by natefoo**

### CODE REVIEW

File: contrib/nagios/check_galaxy.py [hunk 1]
```diff
@@ -15,8 +15,11 @@
 import sys
 import tempfile
 import time
+import urllib2
 import warnings
+from user import home
 
+import warnings
```
Comment (nsoranzo) [inline contrib/nagios/check_galaxy.py]: `warnings` was already imported above.

There are few other flake8 errors at https://travis-ci.org/galaxyproject/galaxy/jobs/199025432

Comment (natefoo) [reply contrib/nagios/check_galaxy.py]: Thanks!

File: contrib/nagios/check_galaxy.py [hunk 2]
```diff
@@ -83,120 +56,108 @@ def usage():
...
-            return(True)
-        except twill.errors.TwillAssertionError as e:
-            tc.get_browser()._browser.set_handle_redirect(True)
-            dprint( "%s is not returning redirect (302): %s" % (url, e) )
-            code = tc.browser.get_code()
-            if code == 502:
-                sys.exit("Galaxy is down (code 502)")
-            return False
+    def delete_history(self):
+        # note, this could cause a history to be created and then deleted.  i don't care.
```
Comment (martenson) [inline contrib/nagios/check_galaxy.py:117]: bliss

Comment (natefoo) [reply contrib/nagios/check_galaxy.py:117]: 'zactly.

**Pull request merged**